.github/workflows/ci.yml

```diff
@@ -80,10 +109,66 @@ jobs:
     with:
       changed-files: ${{ needs.get-changed-files.outputs.changed-files }}
 
+  upload-build-artifacts:
+    name: Deploy Prebuilt Python Builds
+    if: ${{ inputs.kind == 'release' && success() }}
+    uses: ./.github/workflows/deploy-build-action.yml
+    needs:
+      - build-python-package
+      - build
+    with:
+      version: "${{ needs.build-python-package.outputs.version }}"
+    secrets:
+      SSHKEY: "${{ secrets.SSHKEY }}"
```

Comment:
Every build will have `secrets.SSHKEY` available.
We should lock these secrets behind an [environment](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment) which will force a manual step only available to predefined users/teams.

Reply:
This will have to be a TODO for now.